PITCHME.md

```diff
@@ -1,55 +1,60 @@
-# @css[shadow](@color[#ECD290](Cream)<br/>@color[#F15322](City)<br/>@color[#688B8B](Code))
+# @css[shadow](@color[#ECD290](Cream)<br>@color[#F15322](City)<br>@color[#688B8B](Code))
 
 ---
 
-## @css[shadow](@css[ccc-cream](Code) @css[ccc-orange](of) @css[ccc-blue](Conduct))
+## @css[shadow](@css[ccc-cream](Code of Conduct) )
+### @css[shadow](@css[lowercase](@css[ccc-orange](bit.ly/fallx-coc)))
+@css[shadow span-100](@size[.65em](Our festival is dedicated to providing a harassment-free festival experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion (or lackthereof&#41;, or technology choices. We do not tolerate harassment of festival participants in any form. Sexual language and imagery is not appropriate for any festival venue, including talks, workshops, parties, Twitter and other online media. Festival participants violating these rules may be sanctioned or expelled from the festival without a refund at the discretion of the festival organizers.))
 
-@css[shadow](@size[.8em](Our conference is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion or lackthereof, or technology choices. We do not tolerate harassment of conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties, Twitter and other online media.))
+---
+
+<img src="assets/fallx-logo.png" id="fallx-logo" height="300px" />
 
 ---
-@size[6em](@css[ccc-cream shadow](@fa[ticket]))
-<br/>
-@size[3em](@css[shadow](@css[ccc-orange](Patrick) @css[ccc-blue](Dolan)))
 
+# @css[shadow](@css[ccc-cream](Social Media))
+ @size[1.8em](@css[twitter](@fa[twitter] @creamcitycode))
+ <br>
+ @size[1.8em](@css[twitter](#creamcitycode))
+ <hr>
+ @size[1.8em](@css[twitter](@fa[twitter] @fallexperiment))
+ <br>
+ @size[1.8em](@css[twitter](#FallX))
 ---
 
-# @css[ccc-orange shadow](Agenda)
+# @css[shadow](@css[ccc-cream](You Are Here))
 
-Note:
+# @size[2em](@css[ccc-orange](@fa[map-marker]))
 
-Restrooms, Sessions, Daily-Flow (Lunch, Happy Hour, Etc)
+---?image=assets/map.jpg&size=contain&opacity=100
```

Comment:
This syntax I _think_ means "use this as a background image", which we don't need anymore now that I removed all text on the page....it probably could be rendered the way the rest of the big images are rendered, but since I couldn't see it, I didn't know how large/small to scale it. So I left it in this format.